src/utils/debounce.ts

```diff
@@ -0,0 +1,26 @@
+
+export function debounce<T extends (...args: any[]) => void>(fn: T, wait?: number, leading?: boolean) {
```

Comment:
We should make `wait` a required parameter here, i.e. `wait: number`. (Then you should also remove the non-null assertions below: `wait!` -> `wait`.)

Reply:
Or you can set a default value for it like `wait: number = 0`.

Reply:
Thanks for tips ❤️‍🔥